spiderable_server.js

```diff
@@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
 var urlParser = Npm.require('url');
 var crypto = Npm.require('crypto');
 
+var bound = Meteor.bindEnvironment(function(callback) {
```

Comment:
Okay, I'll work around this now.
Next - send new pull request with caching.
One more thing - what do you think about to move on CoffeeScript?